design/bgp-bfd.md

```diff
@@ -0,0 +1,91 @@
+# BFD enablement in BGP
+
+## Summary
+
+The purpose of this enhancement is to use enable [BFD](https://en.wikipedia.org/wiki/Bidirectional_Forwarding_Detection) to be used
+in conjunction with BGP.
```

Comment:
I think we need to specify this is only for the FRR implementation and not the native implementation.

Reply:
Also we will probably need to follow this: https://github.com/metallb/metallb/blob/main/design/0001-frr.md#development-phases

Reply:
> I think we need to specify this is only for the FRR implementation and not the native implementation even though you mention it below.

Ok, I'll add it into the summary as well.

Reply:
> Also we will probably need to follow this: https://github.com/metallb/metallb/blob/main/design/0001-frr.md#development-phases

This feels way more straightforward. What we can do is declare the feature beta somehow, but apart from that the feature will get in all in once I think